docs/conf/portland/2020/hike.rst

```diff
@@ -30,7 +30,7 @@ from the venue to the park, or you can `take transit <https://www.google.com/map
 What to bring
 ~~~~~~~~~~~~~
 
-May in Portland can be interesting. It will probably be in the 60s, with a chance of rain. So, please bring:
+August in Portland can be interesting. It will probably be in the 60s, with a chance of rain. So, please bring:
```

Comment:
THIS IS HILARIOUS

Reply:
Oh yeah, I did kind of a blanket update.